ami/adjust-cgroup-settings.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
+
+# Exclude containers we don't want effected by limits
+EXCLUDES_PATTERN=$(cat <<'EOF' | xargs | sed 's/ /|/g'
+amazon/amazon-ecs-agent
+goodeggs/logspout-goodeggs
+yelp/docker-custodian
+convox/api
+convox/agent
+goodeggs/ranch-api
+EOF
+)
+
+# Build a list of stuff we DO want to effect
+TARGETS=$(docker ps --no-trunc | grep -v 'CONTAINER' | grep -Ev "$EXCLUDES_PATTERN" | awk '{ print $1; }' | xargs)
```

Comment:
how about using `--format` on your `docker ps` to get a stable output format?
https://docs.docker.com/engine/reference/commandline/ps/#formatting